sql/share/errmsg-utf8.txt

```diff
@@ -4769,25 +4769,25 @@ ER_LOCK_OR_ACTIVE_TRANSACTION
         swe "Kan inte utföra kommandot emedan du har en låst tabell eller an aktiv transaktion"
         ukr "Не можу виконати подану команду тому, що таблиця заблокована або виконується транзакція"
 ER_UNKNOWN_SYSTEM_VARIABLE  
-        chi "未知系统变量 '%-.*s'"
-        cze "Neznámá systémová proměnná '%-.*s'"
-        dan "Ukendt systemvariabel '%-.*s'"
-        eng "Unknown system variable '%-.*s'"
-        est "Tundmatu süsteemne muutuja '%-.*s'"
-        fre "Variable système '%-.*s' inconnue"
-        ger "Unbekannte Systemvariable '%-.*s'"
-        geo "უცნობი სისტემური ცვლადი '%-.*s'"
-        hindi "अज्ञात सिस्टम वैरिएबल '%-.*s'"
-        ita "Variabile di sistema '%-.*s' sconosciuta"
-        jpn "'%-.*s' は不明なシステム変数です。"
-        nla "Onbekende systeem variabele '%-.*s'"
-        por "Variável de sistema '%-.*s' desconhecida"
-        rus "Неизвестная системная переменная '%-.*s'"
-        serbian "Nepoznata sistemska promenljiva '%-.*s'"
-        spa "Variable de sistema '%-.*s' desconocida"
-        sw "Tofauti ya mfumo isiyojulikana '%-.*s'"
-        swe "Okänd systemvariabel: '%-.*s'"
-        ukr "Невідома системна змінна '%-.*s'"
+        chi "未知系统变量 '%-.*s'，你是想输入 '%.*s' 吗？"
```

Comment:
I do not think you can edit error messages like this. It will most likely break older binaries reusing the same message file. afaik the correct way is to add a new error message. but please double-check this with the "final" reviewer.